ghost/webmentions/test/InMemoryMentionRepository.test.js

```diff
@@ -76,4 +76,35 @@ describe('InMemoryMentionRepository', function () {
         assert(pageThree.meta.pagination.prev === 2);
         assert(pageThree.meta.pagination.next === null);
     });
+
+    describe(`GetPage`, function () {
```

Comment:
This test doesn't really provide any code coverage but it confirm the behavior we want - that the front end will not receive deleted mentions.